cmd/watch.go

```diff
@@ -102,6 +109,16 @@ func handleWatchEvent(client kit.ThemeClient, asset kit.Asset, event kit.EventTy
 		blue(asset.Key),
 	)
 	if err := perform(client, asset, event, nil); err != nil {
+		notificationMessage("There is a error in your templates check your logs", "Watch", "")
 		stdErr.Printf("[%s] %s", green(client.Config.Environment), err)
 	}
 }
+
+func notificationMessage(message string, title string, iconPath string) {
+	notify = notificator.New(notificator.Options{
+		DefaultIcon: basepath + "/docs/assets/images/shopify-logo.png",
+		AppName:     "ThemeKit",
```

Comment:
I would say this should be "Shopify Themekit" Also there should be a flag to alter this so that tools that want to leverage this like slate can rename it.